core/src/services/config.js

```diff
@@ -91,9 +95,7 @@ class LuigiConfigManager {
    * @returns {boolean} returns true if authorization is enabled. Otherwise returns false.
    */
   isAuthorizationEnabled() {
-    const idpProviderName = this.getConfigValue('auth.use');
```

Comment:
This check is now different, before we were checking `auth.use` and the auth settings, now we are checking only `auth.use`. I think we should still check if the auth settings are also added to the Luigi configuration.

Reply:
The implementation was wrong before, since it never reached the checks vor provider existence and validity inside _Authorization_ component, as it returned too early. As soon as there is some `auth.use` set, we can assume authorization should be used.